superset/models/helpers.py

```diff
@@ -87,14 +87,6 @@ class ImportExportMixin:
 
     __mapper__: Mapper
 
-    @classmethod
-    def _parent_foreign_key_mappings(cls) -> Dict[str, str]:
-        """Get a mapping of foreign name to the local name of foreign keys"""
-        parent_rel = cls.__mapper__.relationships.get(cls.export_parent)
-        if parent_rel:
-            return {l.name: r.name for (l, r) in parent_rel.local_remote_pairs}
-        return {}
-
```

Comment:
This is not being used anywhere, seems like the result of a bad merge because we have an identical class method called `parent_foreign_key_mappings` (no leading underscore).